specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -286,6 +286,17 @@ op getKeyValue is standardOps.ResourceRead<
       format: "csv",
     })
     select?: KeyValueFields[];
+
+    #suppress "@azure-tools/typespec-azure-core/prefer-csv-collection-format" "Pre-existing API contract"
+    @doc("""
+      A filter used to query by tags. Syntax reference:
+      https://aka.ms/azconfig/docs/keyvaluefiltering
+      """)
+    @query({
+      name: "tags",
+      format: "multi",
+    })
+    tags?: string[];
   }> &
     RequestHeadersTrait<{
```

Comment:
The new API version introduces the Cache-Control header. Is there a way to specify the new response header (which may or may not exist depending on the SAS token)?